redisinsight/ui/src/components/monitor/MonitorWrapper.tsx

```diff
@@ -38,18 +41,28 @@ const MonitorWrapper = () => {
 
   return (
     <section className={styles.monitorWrapper} data-testid="monitor-container">
-      <MonitorHeader handleRunMonitor={handleRunMonitor} />
-      <Monitor
-        items={items}
-        error={error}
-        isStarted={isStarted}
-        isPaused={isPaused}
-        isRunning={isRunning}
-        isShowCli={isShowCli}
-        isSaveToFile={isSaveToFile}
-        isShowHelper={isShowHelper}
-        handleRunMonitor={onRunMonitor}
-      />
+      <FeatureFlagComponent
+        name={FeatureFlags.disabledByEnv}
+        enabledByDefault
+        otherwise={(
+          <div style={{ display: 'grid', placeContent: 'center', height: '100%' }}>
+            <EuiTextColor color="accent">Profiler not implemented in this environment.</EuiTextColor>
```

Comment:
I know this shouldn't even be accessible in Cloud right now, but since we're changing it anyway, can we change this to "Profiler not supported" instead? "Not implemented" makes it seem like we forgot to do it.

Reply:
I asked for Olga's input about this, but still have nothing. Will use `not supported` instead